process-cmds.sh

```diff
@@ -202,98 +202,98 @@ inspect-system-ovs-ovn() {
+  kubectl exec -n "${FT_NAMESPACE}" "${TEST_TOOLS_POD}" -- /bin/sh -c "chroot /host /bin/bash -c \"ip addr show\" | cat" >> "${HWOL_VALIDATION_FILENAME}"
   echo "### Command: ip route" >> "${HWOL_VALIDATION_FILENAME}"
-  kubectl exec -n "${FT_NAMESPACE}" "${TEST_TOOLS_POD}" -- /bin/sh -c "chroot /host /bin/bash -c \"ip route\"" >> "${HWOL_VALIDATION_FILENAME}"
+  kubectl exec -n "${FT_NAMESPACE}" "${TEST_TOOLS_POD}" -- /bin/sh -c "chroot /host /bin/bash -c \"ip route\" | cat" >> "${HWOL_VALIDATION_FILENAME}"
   echo "### Command: ip neigh" >> "${HWOL_VALIDATION_FILENAME}"
-  kubectl exec -n "${FT_NAMESPACE}" "${TEST_TOOLS_POD}" -- /bin/sh -c "chroot /host /bin/bash -c \"ip neigh\"" >> "${HWOL_VALIDATION_FILENAME}"
+  kubectl exec -n "${FT_NAMESPACE}" "${TEST_TOOLS_POD}" -- /bin/sh -c "chroot /host /bin/bash -c \"ip neigh\" | cat" >> "${HWOL_VALIDATION_FILENAME}"
   echo "### Command: ethtool -i ${TEST_VF_REP}" >> "${HWOL_VALIDATION_FILENAME}"
-  kubectl exec -n "${FT_NAMESPACE}" "${TEST_TOOLS_POD}" -- /bin/sh -c "chroot /host /bin/bash -c \"ethtool -i ${TEST_VF_REP}\"" >> "${HWOL_VALIDATION_FILENAME}"
+  kubectl exec -n "${FT_NAMESPACE}" "${TEST_TOOLS_POD}" -- /bin/sh -c "chroot /host /bin/bash -c \"ethtool -i ${TEST_VF_REP}\" | cat" >> "${HWOL_VALIDATION_FILENAME}"
   echo "### Command: ethtool -i ovn-k8s-mp0" >> "${HWOL_VALIDATION_FILENAME}"
-  kubectl exec -n "${FT_NAMESPACE}" "${TEST_TOOLS_POD}" -- /bin/sh -c "chroot /host /bin/bash -c \"ethtool -i ovn-k8s-mp0\"" >> "${HWOL_VALIDATION_FILENAME}"
+  kubectl exec -n "${FT_NAMESPACE}" "${TEST_TOOLS_POD}" -- /bin/sh -c "chroot /host /bin/bash -c \"ethtool -i ovn-k8s-mp0\" | cat" >> "${HWOL_VALIDATION_FILENAME}"
   echo "### Command: ethtool -i ovn-k8s-mp0_0" >> "${HWOL_VALIDATION_FILENAME}"
-  kubectl exec -n "${FT_NAMESPACE}" "${TEST_TOOLS_POD}" -- /bin/sh -c "chroot /host /bin/bash -c \"ethtool -i ovn-k8s-mp0_0\"" >> "${HWOL_VALIDATION_FILENAME}"
+  kubectl exec -n "${FT_NAMESPACE}" "${TEST_TOOLS_POD}" -- /bin/sh -c "chroot /host /bin/bash -c \"ethtool -i ovn-k8s-mp0_0\" | cat" >> "${HWOL_VALIDATION_FILENAME}"
   echo "### Command: cat /proc/net/nf_conntrack" >> "${HWOL_VALIDATION_FILENAME}"
-  kubectl exec -n "${FT_NAMESPACE}" "${TEST_TOOLS_POD}" -- /bin/sh -c "chroot /host /bin/bash -c \"cat /proc/net/nf_conntrack\"" >> "${HWOL_VALIDATION_FILENAME}"
+  kubectl exec -n "${FT_NAMESPACE}" "${TEST_TOOLS_POD}" -- /bin/sh -c "chroot /host /bin/bash -c \"cat /proc/net/nf_conntrack\" | cat" >> "${HWOL_VALIDATION_FILENAME}"
   echo "" >> "${HWOL_VALIDATION_FILENAME}"
   echo "## OvS Information" >> "${HWOL_VALIDATION_FILENAME}"
   # Workaround: https://bugzilla.redhat.com/show_bug.cgi?id=1803920
   OPENVSWITCHPID=`kubectl exec -n "${FT_NAMESPACE}" "${TEST_TOOLS_POD}" -- /bin/sh -c "chroot /host /bin/bash -c \"cat /var/run/openvswitch/ovs-vswitchd.pid\""`
```

Comment:
One more up here.

Reply:
So this also works with and without piping to cat.

Reply:
Can keep as is.